juju_verify/verifiers/ceph.py

```diff
@@ -310,28 +324,34 @@ def check_availability_zone(self) -> Result:
                 unit.entity_id for unit in units if unit.entity_id in self.unit_ids
             }
             free_units = min(free_app_units[unit.application] for unit in units)
-            logger.debug("The availability zone %s has %d inactive and %d free units. "
-                         "Trying to remove %s units.", str(availability_zone),
-                         len(inactive_units), free_units, units_to_remove)
+            logger.debug(
+                "The availability zone %s has %d inactive and %d free units. "
+                "Trying to remove %s units.",
+                str(availability_zone),
+                len(inactive_units),
+                free_units,
+                units_to_remove,
+            )
 
             if len(units_to_remove.union(inactive_units)) > free_units:
                 result += Result(
                     Severity.FAIL,
                     f"It's not safe to removed units {units_to_remove} in the "
                     f"availability zone '{availability_zone}'. [free_units="
-                    f"{free_units:d}, inactive_units={len(inactive_units):d}]"
+                    f"{free_units:d}, inactive_units={len(inactive_units):d}]",
```

Comment:
What's the point of the trailing coma here? (honest question :D)

Reply:
I find it nice in a situation where I need to add another value to the list/dict/tuple/.. or another argument for a function. 
You may add a new string value to the list, but you forgot to add a comma there.
e.g.
```python
a = [
  "a",
  "b",
  "c",
  "d"
  "e"
]
```
It's kinda useful, kinda annoying :dagger:

Reply:
I get it when it comes to the lists/dicts but find it weird for classes/functions, especially when their signature does not allow further arguments.

Reply:
Yes, I partially agree, but on the other hand you can change the function signature and then this will change as well :hmm: